src/hooks/useUniversalRouter.ts

```diff
@@ -117,7 +117,7 @@ export function useUniversalRouterSwapCallback(
               }),
               ...analyticsContext,
             })
-            if (tx.data !== response.data) {
+            if (!response.data || response.data.length === 0 || response.data === '0x') {
```

Comment:
we should still log SWAP_MODIFIED_IN_WALLET based on the original criteria. if you don't want that to error, it's fine. just throw the error if the new check fails

Reply:
cool cool